src/survey-template/survey-template.module.ts

```diff
@@ -0,0 +1,8 @@
+import { Module } from '@nestjs/common';
+import { SurveyTemplateService } from './survey-template.service';
+
+@Module({
+  providers: [SurveyTemplateService],
+  exports: [SurveyTemplateService],
```

Comment:
We should mount the controller here
```suggestion
  controllers: [SurveyTemplateController],
  exports: [SurveyTemplateService],
```
https://docs.nestjs.com/modules